operator/v1/0000_70_cluster-network-operator_01_crd.yaml

```diff
@@ -423,7 +423,7 @@ spec:
                         minItems: 1
                         items:
                           type: string
-                          pattern: ^(([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]):[0-9]+$
+                          pattern: (([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])\.){3}([0-9]|[0-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5]):([1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$
```

Comment:
You lost the leading  ^ to match the start of the string... I think you need that.

And this is supposed to match ports 1-65535, right?  If so, the rest of the regex is good.